packages/gatsby/src/bootstrap/load-plugins/__tests__/fixtures/subplugins/node_modules/gatsby-plugin-child-with-main/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "gatsby-plugin-child-with-main",
+  "main": "lib/index.js"
```

Comment:
test case setup for issue described in https://github.com/gatsbyjs/gatsby/issues/33677